docs/software.md

```diff
@@ -0,0 +1,61 @@
+# Software
+
+This page lists general purpose open source software that manipulates HGVS
+expressions. Our intention is to help the community find software that aids in
+the adoption of HGVS Nomenclature. The entries below are provided by the
+authors; the entries are not validated or endorsed by the HVNC. Software packages are listed
+alphabetically.  See [Listing Definitions](#listing-definitions) for an
+explanation of terms. See the [submission requirements and
+instructions](#submissions) for information about submitting to this list.
+
+
+```sh exec="true"
+bin/make-software-table docs/software/*.yml
+```
+
+## Listing Definitions
+
+- **Software Type**: One of:
+    - Library: Software component that is used by developers when building applications
+    - API (e.g., REST): Software that exposes an internet-visible programming interface that is language-independent
+    - Web UI: Software used with a browser window and generally intended for broader audiences
+
+- **License**: Only open-source software with licenses approved by the [Open Source
+  Initiative](https://opensource.org) will be listed. In addition,
+  [well-recognized
+  licenses](https://opensource.org/licenses/?categories=popular-strong-community)
+  will be preferred.
+
+- **Functionality**: Software is annotated as performing one or more of the following functions:
+    - conversion: Does the software convert to/from HGVS and other formats (e.g., VCF, SPDI, VRS)
+    - liftover: Does the software provide *direct* liftover between aligned genomic sequences?
+    - normalization: Does the software normalize variants (e.g., 3' shift) and rewrite (e.g., `ins` → `dup`)
+    - parsing: Does the software parse HGVS expressions into a usable structure for analysis?
+    - projection: Does the software project variants from one sequence to another?
+    - translation: Does the software infer protein consequences from transcript variants?
+    - validation: Does the software provide syntactic validation (i.e., the correct form) and semantic validation (i.e., the parsed structure makes sense)?
+
+## Submissions
+
+### Requirements
+
+The software must:
+
+- Be *primarily* concerned with provided the functionality listed above.
+  Software that merely accepts or emits HGVS expressions as part of their
+  operation (e.g., variant annotation) do not meet this criterion.
+- Meet the license requirements above. Commercial software, including hybrid
+  open-source/commercial software, will not be included.
+- Have a publication in PubMed. This requirement is intended to create a bar
+  that implies review by peers.
```

Comment:
This feels a little artificial. What does it mean to have a PubMed ID? How central a piece of software is to an article can be highly variable. Will we make that distinction? If so, are we effectively becoming peer review? And if that, why require an article at all?

I think it would be better for us to define specific criteria relating to claimed functionality. Perhaps require that any implementations supporting one or more of the above functions pass some standard functionality test for each such function?

Reply:
Well, kinda like it say... it's a bar. I'd like to stay away from debates about functionality until we have a compliance suite. In the absence of that, I think it needs to be self-described.